### ISSUE TITLE
Issue with half opening parens and surrounding bbcode

### ISSUE THREAD

**Issue by ghost** [jmrware/LinkifyURL#1]
Hi,

In this example:

```
[br](bron http://www.example.com/)[br]
```

Linkify parses as:

```
[br](bron [url]http://www.example.com/)[br][/url]
```

while one would expect it to parse

```
[br](bron [url]http://www.example.com/[/url])[br]
```

Is there any way around this? :)

**Comment by jmrware** [comment]
Yes this is a known shortcoming of this version of LinkifyURL. If you read Note 2 on the [LinkifyURL main test page](http://jmrware.com/articles/2010/linkifyurl/linkify.html) you will see other examples of URLs which are incorrectly matched.

I'm looking into another way of coding this that will correct this erroneous matching (but doing this correctly for all cases is quite tricky!) I have noted this limitation as an issue and will be working on it.

Thanks for the report.

**Comment by ghost** [comment]
Thanks for the feedback, I didn't recognize this particular pattern to be incorrectly matched hence the issue :)
